hacep/src/main/java/org/kie/hacep/core/infra/DefaultSessionSnapShooter.java

```diff
@@ -103,23 +93,28 @@ public SnapshotInfos deserialize() {
             }
             consumer.close();
             SnapshotMessage snapshotMsg = bytes != null ? SerializationUtil.deserialize(bytes) : null;
-
+            /* if the Snapshot is ok we use the KieContainer and KieSession from the infos,
+            otherwise (empty system) we will create kiecontainer and kieSession from the envConfig's GAV */
```

Comment:
Where is this logic implemented? From what I can see here the logic is: if snapshotMsg is null, then return null.

Reply:
https://github.com/kiegroup/openshift-drools-hacep/pull/57/files/6473ae81033403cc36581e0ae360c32217242ca1#diff-0e96f193a5e393f196153f1fbee5ef0eR74 I'll remove the comment here